articles.html

```diff
@@ -0,0 +1,391 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="shortcut icon" href="favicon.ico" type="image/x-icon">
+    <link href="styles.css" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/4cce7a8c86.js"></script>
+    <title>WineTime</title>
+</head>
+
+<body>
+
+    <header>
+        <div class="conteiner-stop-link"></div>
+        <section class="header-wrapper--orange">
+            <div class="header-top content-width">
+                <div class="header-top--left">
+                    <i class="fa fa-phone" aria-hidden="true"></i>
+                    <a class="header-top-text" href="tel:+18044 494 0324">044 494 0324</a>
```

Comment:
why different numbers? in tel and inside the link

Reply:
???))
See how that looks in your code
and give me right nomber